test/utils/paths.go

```diff
@@ -64,12 +64,17 @@ func RootDir() (string, error) {
 }
 
 // GetK8sBuildOutputDir returns the build output directory for k8s
-func GetK8sBuildOutputDir() (string, error) {
+// For dockerized build, targetArch (eg: 'linux/arm64', 'linux/amd64') must be explicitly specified
+// For non dockerized build, targetArch is ignored
+func GetK8sBuildOutputDir(isDockerizedBuild bool, targetArch string) (string, error) {
```

Comment:
see also: https://github.com/kubernetes/kubernetes/blob/b2a1855f2c2241bab6ac21e4b1849f74237421af/hack/lib/util.sh#L205

I would recommend either sourcing this script or replicating the logic.

Reply:
Ack, I can potentially follow up in a separate PR. In the bash util, it looks for specific binary, and it checks which locations contains the latest binary build. Whereas for `GetK8sBuildOutputDir()`, it just returns a common build output dir, and rely on the client to check for whether the binary exists.

Reply:
we'll want to minimize skew between how we select binaries for testing, this is what cluster e2e's use typically, we should at least match the search paths. TODO for followup

Reply:
Okay, I can follow up with a different PR for a quick refactor, I think the API would be something like `GetK8sBinary(binary, platform String)` instead of `GetK8sOutputDir`